### PR TITLE
Sous-Chefs adoption

### PR DESCRIPTION
**Pull request by damacus**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,7 +22,11 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-1604
+          - debian-9
+          - debian-10
+          - ubuntu-1804
```
Comment (Stromweld) [inline .github/workflows/ci.yml:27]: What about debian 11?

File: metadata.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+source_url       'https://github.com/sous-chefs/ufw'
+issues_url       'https://github.com/sous-chefs/ufw/issues'
 
 supports 'ubuntu'
-
-source_url 'https://github.com/chef-cookbooks/ufw'
-issues_url 'https://github.com/chef-cookbooks/ufw/issues'
-chef_version '>= 12.15'
+supports 'debian'
+chef_version '>= 15.3'
```
Comment (Stromweld) [inline metadata.rb:13]: Is this just standard that we are only supporting 15.3+? I know this is needed if the cookbook is using modern resources.

Comment (damacus) [reply metadata.rb:13]: Standard pattern at this point. According to sous-chefs.org we should be requiring 16+ but we're definitely not going to need anything from that today.

**Comment by kitchen-porter** [comment]
Released as: [4.0.0](https://github.com/sous-chefs/ufw/releases/tag/4.0.0)

**Pull request merged**